src/Microsoft.Health.Fhir.Shared.Api/Registration/FhirServerServiceCollectionExtensions.cs

```diff
@@ -45,18 +46,23 @@ public static class FhirServerServiceCollectionExtensions
                 options.RespectBrowserAcceptHeader = true;
             });
 
+            services.Configure<ForwardedHeadersOptions>(options =>
+            {
+                options.ForwardedHeaders = ForwardedHeaders.All;
```

Comment:
Are there any security implications to specifying all?

Reply:
That's a good question.

The 3 options are protocol, host, and ip address. We need protocol and ip address since that's what are we logging. But if we are forwarding those anyways, seems like we can forward host too.

We can bring this up during the security meeting on Friday.